### PR TITLE
Switch to Bootstrap 4.

### PR DESCRIPTION
**Pull request by jeroenleenarts**
Still one issue.

The social links at the top are not yet behaving in regards to button color. Not sure yet why. Maybe someone knows?

### CODE REVIEW

**Comment by jeroenleenarts** [comment]
Partial of #469

**Comment by fassko** [comment]
Sorry pressed wrong buton. :D

**Review by fassko — DISMISSED** [review]
I'm not a web developer but looks good to me.

File: css/style.css [hunk 1]
```diff
@@ -99,7 +107,7 @@ a:link {
 }
 
 a:visited {
-   color: var(--link-fg-visited);
+/*   color: var(--link-fg-visited);*/
```
Comment (fassko) [inline css/style.css:110]: Is this needed? :)

**Review by fassko — CHANGES_REQUESTED** [review]
Do we need that comment?

**Review by fassko — APPROVED** [review]
Looks good, thanks!

**Pull request merged**